adaptdl/adaptdl/speedup.py

```diff
@@ -171,31 +194,78 @@ def __call__(self, nodes, replicas, return_local_bsz=False):
         self._mem_speedup[mem_indices] = speedup[ret_indices]
         self._mem_local_bsz[mem_indices] = local_bsz[ret_indices]
 
+        ret_atomic_bsz, ret_accumulation_steps = \
+            self._partition_local_bsz(ret_local_bsz, replicas)
+
         if isscalar:
             ret_speedup = ret_speedup.item()
-            ret_local_bsz = ret_local_bsz.item()
-
-        return ((ret_speedup, ret_local_bsz)
-                if return_local_bsz else ret_speedup)
+            ret_atomic_bsz = int(ret_atomic_bsz.item())
+            ret_accumulation_steps = int(ret_accumulation_steps.item())
+        if return_config:
+            return (ret_speedup, (ret_atomic_bsz, ret_accumulation_steps))
+        else:
+            return ret_speedup
+
+    def _partition_local_bsz(self, local_bsz, replicas):
```

Comment:
I think this method needs an explanation comment.

Reply:
Added